Source/Core/DolphinQt/Utils/Resources.cpp

```diff
@@ -40,10 +40,12 @@ void Resources::Init()
 	m_regions[DiscIO::IVolume::COUNTRY_WORLD].load(GIFN("Flag_Europe")); // Uses European flag as a placeholder
 	m_regions[DiscIO::IVolume::COUNTRY_UNKNOWN].load(GIFN("Flag_Unknown"));
 
-	m_platforms.resize(3);
+	m_platforms.resize(4);
 	m_platforms[0].load(GIFN("Platform_Gamecube"));
 	m_platforms[1].load(GIFN("Platform_Wii"));
 	m_platforms[2].load(GIFN("Platform_Wad"));
+	// TODO: Make this consistent with the other files
+	m_platforms[3].load(GetImageFilename(SL("fileplatform"), QString::fromStdString(File::GetThemeDir(SConfig::GetInstance().theme_name))));
```

Comment:
Just put this into UpdatePixmaps() and model it after the code in there, because that's where the theme-specific images are loaded.